config/locales/hyrax.zh.yml

```diff
@@ -871,7 +867,7 @@ zh:
         visibility_during_embargo: 仅限
         visibility_during_lease: 可使用
       proxy_deposit_request:
-        sender_comment: 注释
         transfer_to: 用户
+        sender_comment: 注释
```

Comment:
Please move `sender_comment` back to its prior location.

Reply:
In English version,` sender_comment` is below `transfer_to`:
https://github.com/samvera/hyrax/blob/master/config/locales/hyrax.en.yml#L881